lib/tasks/maintenance/cleanup_commits.rake

```diff
@@ -17,4 +17,15 @@ namespace :maintenance do
   task cleanup_commits: :environment do
     CommitsCleaner.perform_async
   end
+
+  desc "Cleans up commits that have been deleted from the repository"
+  task reap_deleted_commits: :environment do
+    Commit.find_each do |c|
```

Comment:
I _still think_ it makes more sense to limit this to just commits that haven't completed translation yet. Otherwise once per day Shuttle is going to do be doing thousands of git operations to check that all of the historical commits still exist. WDYT?

Reply:
So in theory:

Day 1: thousands of git operations, thousands of reaped commits
Day 2-∞: Most commits in Shuttle actually exist in the repo, so these would be local git operations only that would succeed before Shuttle would need to fetch the repository

Reply:
Alright ¯\_(ツ)_/¯, it is good that Shuttle checks locally before running git fetch.